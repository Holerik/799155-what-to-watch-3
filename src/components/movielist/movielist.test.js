// movelist.test.js
import React from 'react';
import renderer from 'react-test-renderer';
import MovieList from './movielist.jsx';

const filmsInfo = [
  {
    id: 91,
    title: `The Commuter`,
    poster: `img/commuter.jpg`,
    altPoster: `commuter poster`,
    src: `https://media.w3.org/2010/05/sintel/trailer.mp4`,
    preview: `https://media.w3.org/2010/05/sintel/trailer.mp4`,
  }
];

const mouseClickHandler = jest.fn();
const setActiveItem = () => {};

it(`<MoveList /> should render small movie cadr list`, () => {
  const tree = renderer
    .create(<MovieList
      tabItems={filmsInfo}
      stateItem={0}
      setActiveItem={setActiveItem}
      mouseClickHandler={mouseClickHandler}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
