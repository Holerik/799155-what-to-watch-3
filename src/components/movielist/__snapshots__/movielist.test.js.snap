// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MoveList /> should render small movie cadr list 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <div
      className="small-movie-card__image"
      id="91"
      onClick={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
    >
      <img
        alt="commuter poster"
        height="175"
        src="img/commuter.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
      onClick={[Function]}
    >
      <a
        className="small-movie-card__link"
        href="/moviecard-full"
      >
        The Commuter
      </a>
    </h3>
  </article>
</div>
`;
